### DISCUSSION TITLE
Cefsharp offscreen produce single file

### DISCUSSION THREAD

**Discussion by Zer0bytte** [cefsharp/CefSharp#4528] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

106

### Operating System

Windows 10

### Architecture

x64

### .Net Version

.net 7

### Implementation

OffScreen

### Reproduction Steps

NA

### Expected behavior

NA

### Actual behavior

![image](https://github.com/cefsharp/CefSharp/assets/85482481/f34de1c6-fc07-4663-bf37-fd0642304397)

When publishing single file program not start and i have debugged it in dnspy shows that exception can't load that file

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by amaitland** [comment]
Please stop opening issues for support questions. In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask support related questions.

> * I have searched both open/closed issues, no issue already exists.

There are multiple issues that cover the topic of publishing a single file. There's a summary in https://github.com/cefsharp/CefSharp/discussions/4444#discussioncomment-5527409
